pkgs/by-name/co/code-cursor/package.nix

```
@@ -5,7 +5,9 @@
   appimageTools,
   makeWrapper,
   writeScript,
+  undmg,
 }:
+
 let
   pname = "cursor";
   version = "0.44.11";
```

```suggestion
  version = "0.44.11";
  
  inherit (stdenvNoCC) buildPlatform;
```

Tidying up the code.